lib/main.rb

```diff
@@ -1,11 +1,49 @@
-require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative 'app'
 
-person = Person.new(22, 'maximiliano')
+def print_menu
+  puts
+  puts 'Please choose an option by entering a number:'
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person id'
+  puts '7 - Exit'
+  puts 'Enter your choice:'
+end
 
-capitalized_person = CapitalizeDecorator.new(person)
-puts "Capitalized name: #{capitalized_person.correct_name}"
+def handle_choice(choice, app)
+  case choice
+  when 1
+    app.list_books
+  when 2
+    app.list_people
+  when 3
+    app.create_person
+  when 4
+    app.create_book
+  when 5
+    app.create_rental
+  when 6
+    app.list_rentals
+  when 7
+    return false
```

Comment:
- [ ] Again here as requested kindly ensure that the user is able to list rentals. I would also suggest that since the user is currently unable to create a student, once you fix that you check if a student can create a rental as well.
![image](https://github.com/hajnaloltyan/school-library/assets/81932478/14a85c1f-8f3d-4af9-a144-b95cebaefce6)

The error message you're seeing indicates that the person variable is not defined in the list_rentals method of your Ruby application.

Reply:
**[REQUIRED CHANGES]**

- Great job addressing the changes, however, as pointed by the previous reviewer here, when the user tries the 
`6 - List all rentals for a given person id`, after choosing the ID, it throws the error below. I think this is an oversight from you as you didn't test all the options before submitting for another review. Please, try to fix it. :clap: 


```
r">>]> (NameError)

      rentals = @rentals.select { |rental| rental.person == person }
                                                            ^^^^^^
Did you mean?  person_id
        from /home/zieeco/Desktop/school-library/lib/app.rb:47:in `select'
        from /home/zieeco/Desktop/school-library/lib/app.rb:47:in `list_rentals'
        from main.rb:29:in `handle_choice'
        from main.rb:42:in `block in main'
        from main.rb:38:in `loop'
        from main.rb:38:in `main'
        from main.rb:49:in `<main>'
```